ross/shaft_element.py

```diff
@@ -47,7 +47,8 @@ class ShaftElement(Element):
     odr : float, pint.Quantity, optional
         Outer diameter of the element at the right position (m).
         Default is equal to odl value (cylindrical element).
-    L : shaft length (m)
+    L : float
+        Shaft length (m).
```

Comment:
Good morning @gutilacerda! In fact, the `L` parameter can be passed as a `pint.Quantity` as well, similar to `odr`. 

Additionally, as it is the first parameter in the `__init__` method, maybe it is better if it is the first in the documentation.

Reply:
Done!